iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -437,7 +437,7 @@ public String identifier(String tableIdentifier) {
       }
 
       Assert.assertTrue(location.delete());
-      return location.toString();
+      return "file://" + location;
```

Comment:
Why was this needed?

Reply:
If the table location does not match the file locations then we had a problem matching the `alias` to the `path`, and failed to add the configuration values to the HiveConf.

We need to use the full path in the table location to match the file names.